dependencies.go

```diff
@@ -1,4 +1,4 @@
-package dependencies
```

Comment:
This file should be inside the internal folder and inside dependancies folder and the package should be dependancies as well.